src/Playwright.Tests/ScreencastTests.cs

```diff
@@ -143,5 +142,4 @@ public async Task ShouldCaptureStaticPageInPersistentContext()
             Assert.NotEmpty(new DirectoryInfo(tempDirectory.Path).GetFiles("*.webm"));
         }
     }
-    */
```

Comment:
Just to check - you've created an issue for this, right? To not have commented out code in there?

Reply:
I'm removing the commented code :)